src/codecoverage/bot/tools/covdir_gen.py

```diff
@@ -59,7 +53,8 @@ def list_commits(maximum=None, unique=None, skip_commits=[]):
             dates.add(week)
 
             # Convert git to mercurial revision
-            commit['mercurial'] = github.git_to_mercurial(commit['commitid'])
+            # TODO: use releng mapper here
+            commit['mercurial'] = None  # convert(commit['commitid'])
```

Comment:
You can use libmozdata new module

Reply:
Done, thanks for your implementation on libmozdata :smiley: